### PR TITLE
Pick up files from S3 bucket

### PR DESCRIPTION
**Pull request by Floppy**
For  https://trello.com/c/kaWFjQlO

### CODE REVIEW

**Comment by Floppy** [comment]
I suspect the next step is to use something like https://docs.python.org/3/library/tarfile.html to open up the tar.gz file (attached to the trello ticket) and read the data direct from that.

File: lambda_function.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+import urllib3
 
 def handler(event, context):
     decoder = json.decoder.JSONDecoder()
     message = decoder.decode(event['Records'][0]['Sns']['Message'])
     print("Consignment reference:", message["consignment-reference"])
+    http = urllib3.PoolManager()
+    metadata_file = http.request('GET', message["s3-folder-url"])
+    metadata = decoder.decode(metadata_file.data.decode('utf-8'))
+    print(metadata["bagit-info"]["Internal-Sender-Identifier"])
```
Comment (lozette) [inline lambda_function.py]: Is this debug or a necessary part of the code?

**Review by lozette — COMMENTED** [review]
One question!

File: lambda_function.py (hunk 1 above)
Comment (LewisDaleUK) [reply lambda_function.py]: I think you might be looking at an old diff, that's been removed now

Comment (lozette) [reply lambda_function.py]: 🤦

**Review by lozette — APPROVED** [review]

**Pull request merged**